biomaj_download/download/direct.py

```diff
@@ -91,24 +98,21 @@ def list(self, directory=''):
             except Exception:
                 self.crl.setopt(pycurl.URL, file_url.encode('ascii', 'ignore'))
             self.crl.setopt(pycurl.URL, file_url)
-            self.crl.setopt(pycurl.OPT_FILETIME, True)
-            self.crl.setopt(pycurl.NOBODY, True)
 
             # Note that very old servers may not support the MDTM commands.
             # Therefore, cURL will raise an error (although we probably can
             # download the file).
             try:
                 self.crl.perform()
-                errcode = self.crl.getinfo(pycurl.RESPONSE_CODE)
-                if int(errcode) not in self.ERRCODE_OK:
+                errcode = int(self.crl.getinfo(pycurl.RESPONSE_CODE))
+                if errcode != 350 and errcode not in self.ERRCODE_OK:
```

Comment:
If code is 350 we should not try to get time/size info

Reply:
In this case, `350` indicates success (this is the return of the last `REST` command). If the code is different, we raise an exception because this probably indicates an error.